drag_and_drop_v2/templates/html/drag_and_drop_edit.html

```diff
@@ -21,6 +21,16 @@
                       <span class="sr">{{ help_texts.show_title }}</span>
                     </label>
 
+                    <label class="h3" for="problem-mode" title="{{ help_texts.mode }}">{% trans "Problem mode" %}</label>
+                    <select id="problem-mode">
```

Comment:
Take care with using ids like this. We have to make sure that ids are unique on the page, even if there are multiple instances of the xblock within the unit. In this particular case it is OK because we only allow a single component to be edited at a time, but seeing an id like this still makes me cringe.